content/docs/intro/cloud-providers/alicloud/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: AliCloud
```

Comment:
```suggestion
title: Alibaba Cloud
```

?

If so, applies throughout this page and the setup page.

Reply:
Nice catch! Thanks :) Just pushed the fix to update it all